examples/README.md

```diff
@@ -12,3 +12,12 @@
 cd ebpf/examples/
 go run -exec sudo [./kprobe, ./uretprobe, ./tracepoint, ...]
 ```
+
+## How to compile
```

Comment:
This kind of duplicates the section above, is it necessary?

Reply:
Yeah, I will merge them.